src/operations.cc

```diff
@@ -82,6 +82,70 @@ namespace sharp {
   }
 
   /*
+    Cutout src over dst with given gravity.
+   */
+  VImage Cutout(VImage mask, VImage dst, const int gravity) {
+    using sharp::CalculateCrop;
+    using sharp::HasAlpha;
+    using sharp::MaximumImageAlpha;
+
+    bool maskHasAlpha = HasAlpha(mask);
+    bool dstHasAlpha = HasAlpha(dst);
```

Comment:
I think `dstHasAlpha` will always be `true` as there's logic elsewhere to ensure the presence of an alpha channel.